packages/cli/src/services/cache.service.ts

```diff
@@ -5,9 +5,12 @@ import type { MemoryCache } from 'cache-manager';
 import type { RedisCache } from 'cache-manager-ioredis-yet';
 import { jsonStringify } from 'n8n-workflow';
 import { getDefaultRedisClient, getRedisPrefix } from './redis/RedisServiceHelper';
+import { MetricsService } from '../metrics';
 
 @Service()
 export class CacheService {
```

Comment:
Should we switch to pattern that every service that needs to track metrics, either extends EventEmitter, or has a readonly property that's an event emitter, so that the `MetricsService` can listen to those, instead of us having to tightly couple metrics service everywhere.